pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -541,7 +542,10 @@ func testHTTPSClassification(t *testing.T, tr *Tracer, clientHost, targetHost, s
 				// The server might not be ready to accept connection just yet, so we
 				// try until it starts accepting them.
 				require.EventuallyWithT(t, func(c *assert.CollectT) {
-					makeRequest(c, client, requestURL)
+					for i := 0; i < 5; i++ {
```

Comment:
Why is another retry loop needed inside the EventuallyWithT which in itself is a retry loop?